plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/statistics/DeltaLakeJsonFileStatistics.java

```diff
@@ -141,7 +142,15 @@ private Optional<Object> deserializeStatisticsValue(DeltaLakeColumnHandle column
         if (!columnHandle.isBaseColumn()) {
             return Optional.empty();
         }
-        Object columnValue = deserializeColumnValue(columnHandle, statValue, DeltaLakeJsonFileStatistics::readStatisticsTimestamp, DeltaLakeJsonFileStatistics::readStatisticsTimestampWithZone);
+
+        Object columnValue;
+        try {
+            columnValue = deserializeColumnValue(columnHandle, statValue, DeltaLakeJsonFileStatistics::readStatisticsTimestamp, DeltaLakeJsonFileStatistics::readStatisticsTimestampWithZone);
+        }
+        catch (TrinoException e) {
+            log.warn("Cannot deserialize column value, skipping statistics: %s", e.getMessage());
```

Comment:
Could you add test case with unsupported statistics types?

Reply:
Test case added! 👍